conf/k8sCorefile

```diff
@@ -0,0 +1,13 @@
+# Serve on port 1053
+.:1053 {
```

Comment:
I like adding examples, but I assume this is just used for testing (and checking in was the easy way?) :-)